binaries/cuprated/src/main.rs

```diff
@@ -61,6 +61,9 @@ fn main() {
     // Initialize logging.
     logging::init_logging(&config);
 
+    //Printing configuration
+    info!("{config}");
```

Comment:
```suggestion
    // Printing configuration
    ```suggestion
    info!("========== CONFIGURATION ==========\n{config:#?}\n===================================");
```
```